tests/unit/v1/helpers.py

```diff
@@ -129,6 +129,7 @@ def _get_coordinate(
 
 
 def _get_all_coordinates(dataset: Dataset) -> list[Coordinate]:
+    """Get all coordinates from the dataset."""
```

Comment:
```suggestion
    """Get all Coordinates from the Dataset."""
```
Proper noun nit-pick

Reply:
Fixed